librz/core/cmd_debug.c

```diff
@@ -5154,11 +5196,6 @@ RZ_IPI int rz_cmd_debug(void *data, const char *input) {
 		rz_core_cmd_help(core, help_msg_d);
 		break;
 	}
-	if (follow > 0) {
-		ut64 pc = rz_debug_reg_get(core->dbg, "PC");
-		if ((pc < core->offset) || (pc > (core->offset + follow))) {
-			rz_core_seek_to_register(core, "PC", false);
-		}
-	}
+	dbg_follow_seek_register(core);
```

Comment:
You are right, when I tried bringing it back, that test seems to work.
I'll add it back. There's no other hacks to this, is there?

Reply:
I hope not :D

Reply:
Tests pass, except clang-format. And that's because it couldn't fetch something properly. And I hope mine's clearly formatted.

You can probably merge the dev branch and that'll probably fix the failed test.